Tests/IntegrationTests/contentModule.js

```diff
@@ -45,6 +45,7 @@ async function goToPage(t, pageTitle) {
 
 fixture`Content Module`
     .beforeEach(async t => {
+        await t.setNativeDialogHandler(() => true);
```

Comment:
It's like clicking Ok in this state like it's written there.
And yes, waiting would be the better approach.

Reply:
Yep, because otherwise we would loose the ability to spot regressions in side-effects of ajax requests, or even brokenness of this logic itself.

Reply:
I click false now and wait for 1 second, is this better?

Reply:
@mstruebing uhm no, that's even worse I think. Let's try to figure out what is actually going on. So we have a test, that at the end of the test does some ajax request, and the test doesn't wait for it to finish, right?
In this case the better way to fix it would be to wait for ajax requests to finish here in `afterEach`: https://github.com/neos/neos-ui/blob/master/Tests/IntegrationTests/contentModule.js#L52
It could be done in the similar fashion to `waitForIframeLoading`.
Makes sense?

Reply:
Doing it this way, only one failing test-case left :)